src/rest-client.ts

```diff
@@ -727,6 +749,9 @@ export class RestClient extends BaseRestClient {
     return this.getPrivate('/api/v5/trade/fills-history', params);
   }
 
+  /**
+   * @deprecated - use POST /api/v5/account/bills-history-archive
```

Comment:
Once the fn name is decided, I'd rephrase this to refer to the fn name instead of endpoint